clinica/cmdline.py

```diff
@@ -226,7 +227,8 @@ def execute():
         T1VolumeCreateDartelCLI(),
         T1VolumeRegisterDartelCLI(),
         T1VolumeDartel2MNICLI(),
-        T1VolumeParcellationCLI()
+        T1VolumeParcellationCLI(),
+        T1LinearCLI()
```

Comment:
It would be better to put `T1LinearCLI` between T1FreeSurferLongitudinalCLI() and DwiPreprocessingUsingPhaseDiffFieldmapCli(). When typing `clinica run`, `t1-linear` will displayed before `dwi-preprocessing` in the list of pipelines.

`pipelines` contains main pipelines sorted by modalities. 'Advanced' pipelines are displayed at the end of the list.